README.adoc

```diff
@@ -22,6 +22,11 @@ List of tech companies to help developers to find jobs.
 |Android, Java, Spring, Go, JavaScript, AngularJS, NodeJS, AWS, Machine Learning
 |https://www.augmedix.com/[Website]
 
+|Braincraft Limited
```

Comment:
@mmmim24 I didn't find any difference between #24 and #25

Reply:
I didn't find any either. Don't know why the #24 was closed.